examples/Virus_on_Network/requirements.txt

```diff
@@ -0,0 +1,2 @@
+networkx>=2.0
+mesa
```

Comment:
@rht this line is specifically added in the move -- because this model was failing.

Reply:
I was about to say that this should have been pinned to `mesa==1.x`, because people might be cloning this repo and forget to update the repo when it has been changed to be compatible with 2.0 instead of 1.x. But the diff in this PR is huge, and any small iterations on top of it will slow down the process. And so, I'm merging now.